src/commands/api/validate.js

```diff
@@ -23,7 +24,7 @@ class ValidateCommand extends BaseCommand {
     }
     this.log(validationResultsStr)
     
-    if (hasCritical) this.exit(1)
+    if (hasCritical && flags['fail-on-critical']) this.exit(1)
     this.exit(0)
```

Comment:
`-c, --fail-on-critical  Exit with error code 1 if there are critical standardization errors present`
README says about error code 1 but her code 0 is used